File: problems/98/Main.cpp
#include <iostream>
#include <cstdio>
#include <algorithm>
#include <cstring>
#include <string>
#include <cctype>
#include <stack>
#include <queue>
#include <list>
#include <vector>
#include <map>
#include <sstream>
#include <cmath>
#include <bitset>
#include <utility>
#include <set>
#include <numeric>
#include <ctime>
#include <math.h>

using namespace std;

class Solution {
	public:
		vector<int> plusOne(vector<int> &A);
};

//--SPLIT--

//-vector<int> Solution::plusOne(vector<int> &A) {
//-}

//--SPLIT--
vector<int> Solution::plusOne(vector<int> &A) {
	return A;
}

//--SPLIT--
int main() {
	int testcases;
	cin >> testcases;
	for (int t = 0; t < testcases; t++) {
		int n; cin >> n;
		vector<int> A(n);
		for (int i = 0; i < n; i++) {
			cin >> A[i];
		}
		Solution solution;
		vector<int> result = solution.plusOne(A);
		for (int i = 0; i < result.size() ;i++) {
			cout << result[i] << " ";
		}
		cout << "\n";
	}
	return 0;
}

File: problems/98/Main.py
import math

#--SPLIT--
#-class Solution:
	# @param A : list of integers
	# @return a list of integers
#-	def plusOne(self, A):

#--SPLIT--
class Solution:
	def plusOne(self, A):
		return A


#--SPLIT--

if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        A = [int(l) for l in raw_input().split(" ")][1:]
        solution = Solution()
        print(solution.plusOne(A))